website/www/site/content/en/documentation/io/connectors.md

```diff
@@ -0,0 +1,940 @@
+---
+title: "I/O Connectors"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# I/O Connectors
+
+Apache Beam I/O connectors provide read and write transforms for the most popular data storage systems so that Beam users can benefit from native optimised connectivity.  With the available I/Os, Apache Beam pipelines can read and write data from and to an external storage type in a unified and distributed way.
+
+## Built-in I/O Connectors
+
+This table provides a consolidated, at-a-glance overview of the available built-in I/O connectors.
+<div class="table-container-wrapper">
+  <table class="table table-bordered">
+  <tr>
+    <th>Connector Name</th>
+    <th>Source Supported</th>
+    <th>Sink Supported</th>
+    <th>Java</th>
+    <th>Python</th>
+    <th>Go</th>
+    <th>Batch Supported</th>
+    <th>Streaming Supported</th>
+  </tr>
+  <tr>
+    <td>FileIO</td>
+    <td class="present">✔</td>
+    <td class="present">✔</td>
+    <td class="present">
+      ✔
+      <a href="https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/FileIO.html">native</a>
```

Comment:
Could you please change the collor for the all `native` to the green color?
<img width="97" alt="image" src="https://user-images.githubusercontent.com/26118465/158114910-f81ab02f-6829-4772-bfd8-f97a19c85446.png">

Reply:
done